src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -53,12 +53,20 @@
     (normalize-input current v)
     current))
 
+(defn find-possible-networks
```

Comment:
can you mark this private?

Reply:
consider a name like `find-affordable-networks`, it's more suggestive that this "find" is based on the balance? although still feels like there is something better :)

Reply:
> although still feels like there is something better

What do you mean by this?

This function is finding the networks that has balance greater than or equal to the input value to be shown while loading the routes